### DISCUSSION TITLE
Template issues

### DISCUSSION THREAD

**Discussion by nunes3000** [joaomlourenco/novathesis#188] [Q&A] [answered]
Hello professor,
I have some modifications which I would like to do to the thesis template.
I would like to know if it is possible:
1-To change the name "Bibiolgraphy" to "References";
2-To remove the list of figures, the list of tables, Acronyms, etc... from the Contents list (This last would start with Chapter 1 (Introduction));
3-On the Copyright page after the cover page to change the name <UNIVERSITY> to  NOVA Sci. Tech. (FCT NOVA).

I would like to know where can I do such modifications...
Best regards

Ricardo Nunes

**Comment by joaomlourenco** [comment]
1. Use `\addto\captionsLANG{\renewcommand{\bibname}{References}}`  replacing LANG with “english” or ”portuguese”.
2. In the latest version of the template, use "\ntsetup{printfrontmatter=false}"
3. You **should not** change the name of the School in the template.  When the new covers are made official, the template will support them.

**Comment by nunes3000** [comment]
Thank you for the fast answer,
1-I tried to use the command which you suggested inside the "fix-babel" file like this:

![image](https://user-images.githubusercontent.com/84328491/120210779-16af4e00-c228-11eb-9d8c-063dece46ee2.png)
However, the name "Bibliography" still continues..
Do you know what could be the problem?

2- I write that command on the "6_files.tex" file and all the contents list and list of figure etc.. disappeared.  I just wanted the strings "list of figures", "list of tables", "Acronyms", etc... removed from the Contents list without actually delete all the lists. Just like this:
![image](https://user-images.githubusercontent.com/84328491/120211605-064ba300-c229-11eb-971d-64f251adf38a.png)

It is possible to achieve a list of contents like this?

3-Sorry, part of the question was deleted I do not know why. On the Copyright page after the cover page to change the name "UNIVERSITY" to "NOVA Sci. Tech. (FCT NOVA)":
![image](https://user-images.githubusercontent.com/84328491/120211865-4874e480-c229-11eb-8a21-6c8a7ada5ecf.png)


Best regards

Ricardo Nunes

**Comment by joaomlourenco** [comment, marked as answer]
1 — **Do not** change the files in NOVAthesis folder…  never ever!!! 
In file "7_packages.tex", add:
```latex
\DefineBibliographyStrings{english}{%
  bibliography = {WHATEVER YOU WANT},
  references = {WHATEVER YOU WANT},
}
```

2 — You want to remove entries from the Contents page?
To remove the entries related to glossaries add `\glstocfalse` to "7_packages.tex".
To remove the list of figures and the list of tables, in file "8_listof.tex" replacce
```latex
\netaddlistof{listoffigures}      % The List of Figures. Comment to omit.
\netaddlistof{listoftables}       % The List of Tables. Comment to omit.
````
with
```latex
\netaddlistof{listoffigures*}      % The List of Figures. Comment to omit.
\netaddlistof{listoftables*}       % The List of Tables. Comment to omit.
````
3 — **DO NOT** change the name of the School or of the University in the thesis/dissertations until you are told to do so by a norm from the Dean (diretor).
But you have a version that had a bug… either update to the latest version (**highly recommended**) or just fix the file `NOVAthesisFiles/Schools/nova/univ-defaults.ldf` with
```latex
\university(pt):={Universidade NOVA de Lisboa}
\university(en):={NOVA University Lisbon}
\university(fr):={NOVA University Lisbon}
\university(it):={NOVA University Lisbon}
\university(logo):={nova_emblema}
\university(logo,neg):={nova_emblema_negativo}
```

**Reply by leoneldca** [reply]
Olá João,
Eu tentei o mesmo para traduzir o título da lista de Acrônimos mas não funcionou, pois mesmo o documento estando em Português, esta página sempre fica com o título Acronyms, em inglês.
Estou utilizando o template para a UMinho.

O que tentei foi:
\DefineAcronymsStrings{portuguese}{%
  acronyms = {Acrônimos}
}
Não resultou
Pode me ajudar nisso também?

Obrigado e parabéns pelo trabalho desenvolvido

**Reply by b-pereira** [reply]
O `\DefineBibliographyStrings{english}` é um comando do BibLaTex [Using \DefineBibliographyStrings (getting undefined)?](https://tex.stackexchange.com/questions/242267/using-definebibliographystrings-getting-undefined)

Para ~~um _quick fix_ para~~ *Acrónimos* ~~(pode ser que o @joaomlourenco tenha uma solução mais elegante)~~ (não existe solução "elegante")

```tex
\renewcommand*{\acronymname}{Lista de Acrónimos}%
```
ou para o mais correto:

```tex
\renewcommand*{\acronymname}{Siglas e Acrónimos}%
```
onde HTTPS é uma sigla e LAN um acrónimo.

De uma forma geral:

```tex
\renewcommand*{\glossariescaptionsportuges}{%
\renewcommand*{\glossaryname}{Gloss\’ario}%
\renewcommand*{\acronymname}{Lista de Acrónimos}%
\renewcommand*{\entryname}{Nota\c c\~ao}%
\renewcommand*{\descriptionname}{Descri\c c\~ao}%
\renewcommand*{\symbolname}{S\’imbolo}%
\renewcommand*{\pagelistname}{Lista de P\’aginas}%
\renewcommand*{\glssymbolsgroupname}{S\’imbolos}%
\renewcommand*{\glsnumbersgroupname}{N\’umeros}%
}%
```

**Reply by leoneldca** [reply]
Olá Bruno, 
Eu adicionei o comando no arquivo 7_packages.tex e funcionou corretamente. 
Apesar de que, penso que provavelmente deva existir algum local onde se cadastre as traduções de para cada uma das línguas disponíveis. Mas não encontrei. 

O que importa é que funcionou. 

Obrigado e bom dia
